.do/app.yaml

```diff
@@ -0,0 +1,55 @@
+name: sample-django
+services:
+- name: server
+  github:
+    repo: digitalocean/sample-django
+    branch: static
+    deploy_on_push: true
+  run_command: python manage.py runserver 0.0.0.0:${PORT:-8080}
```

Comment:
Since the sample app uses gunicorn I would  change this command to use `gunicorn`. The runserver command is not recommended to be run in production. It also removes any change of the app utilizing more than one thread.

Reply:
Good call — updated!